.changeset/three-walls-perform.md

```diff
@@ -0,0 +1,13 @@
+---
+"gatsby-source-strapi": major
+---
+
+BREAKING CHANGES:
+
+- This plugin now assumes Strapi 5 by default.
```

Comment:
Let's show folks how to edit their config to account for this change

Reply:
i've added a little extra